05-Jason-Luo/wk01/4-thu/js-homework-4.js

```diff
@@ -0,0 +1,74 @@
+// Given the following a rectangle object like the one below, write the following functions which accept a rectangle object as an argument:
+
+// isSquare - Returns whether the rectangle is a square or not
+// area - Returns the area of the rectangle
+// perimeter - Returns the perimeter of the rectangle
+
+
+
+const areaSquare = function (length, width) {
+    const area = length * width;
+    console.log(`Area = ${area}`);
```

Comment:
```suggestion
    return `Area = ${area}`;
```
Make sure you `return` a value when asked to.